doc/whats-new.rst

```diff
@@ -32,7 +32,14 @@ Bug fixes
 ~~~~~~~~~
 
 - :py:func:`merge` with ``combine_attrs='override'`` makes a copy of the attrs (:issue:`4627`).
-
+- By default, when possible, xarray will now always use values of type ``int64`` when encoding
```

Comment:
Should we warn users that this fix may result in a different `units` attribute compared to xarray < 0.16.3?

Reply:
I think that makes sense.  I added a note to the "breaking changes" section.  This particular change impacts cftime encoding/decoding as well.